ogr/utf8.h

```diff
@@ -1422,9 +1422,11 @@ utf8_constexpr14_impl utf8_int32_t utf8lwrcodepoint(utf8_int32_t cp) {
     case 0x01ac:
       cp = 0x01ad;
       break;
+#if 0
```

Comment:
leftover?

Reply:
I had left that as a way to trace back the change w.r.t the upstream project, but I just submitted the removal upstream in https://github.com/sheredom/utf8.h/pull/121. I'll delete that code once upstream has integrated the change

Reply:
done